client/src/plugins/version-info/ReleaseInfo.js

```diff
@@ -44,12 +44,19 @@ export function ReleaseInfo(props) {
     <div className={ css.ReleaseInfo }>
       <ul className="dashed">
         <li>
-          <h4>Edit FEEL expressions in the properties panel</h4>
-          This release adds syntax highlighting and autocomplete for FEEL expressions to the properties panel.
+          <h4>Support for extension properties in Camunda Platform 8</h4>
+          With this release, we added generic extension properties supported by Camunda Platform 8.1.
         </li>
         <li>
-          <h4>Diagram errors in the properties panel</h4>
-          All diagram errors are shown in the properties panel as of this release. Open the error panel for the errors to be shown.
+          <h4>Diagram errors on the canvas</h4>
+          Diagram errors are shown on the canvas as of this release. Click on an overlay to display individual element errors in the properties panel.
```

Comment:
Clicking the overlay doesn't always show an entry in the properties panel (e.g. element type not supported).

How about: _Diagram errors are now shown on the canvas. Click on an error to show details._

Reply:
👍